src/NuGet.Services.Metadata.Catalog.Monitoring/NuGet.Services.Metadata.Catalog.Monitoring.csproj

```diff
@@ -84,8 +84,18 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="Model\IPackageRegistrationMetadataResource.cs" />
+    <Compile Include="Model\PackageRegistrationIndexMetadata.cs" />
+    <Compile Include="Model\PackageRegistrationLeafMetadata.cs" />
+    <Compile Include="Model\PackageRegistrationMetadataResourceV2Feed.cs" />
+    <Compile Include="Model\PackageRegistrationMetadataResourceV3.cs" />
+    <Compile Include="Providers\PackageRegistrationMetadataResourceV2FeedProvider.cs" />
+    <Compile Include="Providers\PackageRegistrationMetadataResourceV3Provider.cs" />
     <Compile Include="Utility\CommonLogger.cs" />
```

Comment:
Not sure how necessary that is--I'm doing it here because the specific classes that I'm following the convention for are extensions of the `NuGet.Core` APIs.